app/templates/people/person/personal-information/index.hbs

```diff
@@ -48,65 +48,108 @@
+                  </AuLink>
                 </:content>
               </Item>
-            {{/if}}
-            <Item>
-              <:label>GSM nummer</:label>
-              <:content />
-            </Item>
-          </:right>
-        </Card.Columns>
-      </:card>
-    </DataCard> --}}
+              {{#if position.primaryContact.email}}
+                <Item>
+                  <:label>E-mail</:label>
+                  <:content>
+                    {{position.primaryContact.email}}
+                  </:content>
+                </Item>
+              {{/if}}
+              {{#if position.primaryContact.contactAddress.fullAddress}}
+                <Item>
+                  <:label>Adres</:label>
+                  <:content>
+                    {{position.primaryContact.contactAddress.fullAddress}}
+                  </:content>
+                </Item>
+              {{/if}}
+              {{#if position.primaryContact.contactAddress.province}}
+                <Item>
+                  <:label>Province</:label>
+                  <:content>
+                    {{position.primaryContact.contactAddress.province}}
+                  </:content>
+                </Item>
+              {{/if}}
+            </:left>
+            <:right as |Item|>
+              {{#if position.role}}
+                <Item>
+                  <:label>Positie</:label>
+                  <:content>
+                    {{#if (eq position.type "minister")}}
+                      <AuLink
+                        @route="people.person.positions.minister"
+                        @models={{array @model.person.id position.id}}
+                      >
+                        {{position.role}}
+                      </AuLink>
+
+                    {{else}}
+
+                      <AuLink
+                        @route="people.person.positions.mandatory"
+                        @models={{array @model.person.id position.id}}
+                      >
+                        {{position.role}}
+                      </AuLink>
+                    {{/if}}
+                  </:content>
+                </Item>
+              {{/if}}
+
+              {{#if position.primaryContact.telephone}}
+                <Item>
+                  <:label>Primair telefoonnummer</:label>
+                  <:content>
+                    {{position.primaryContact.telephone}}
+                  </:content>
+                </Item>
+              {{/if}}
+              {{#if position.secondaryContact.telephone}}
+                <Item>
+                  <:label>Secundair telefoonnummer</:label>
+                  <:content>
+                    {{position.secondaryContact.telephone}}
+                  </:content>
+                </Item>
+              {{/if}}
+            </:right>
+          </Card.Columns>
+        </:card>
+      </DataCard>
+    {{/each}}
+    {{!-- todo when posities list pages is finished, uncomment this}}
+      <div class="flex items-start">
```

Comment:
I think you can use the `<AuHelpText>` component here so you don't need the custom styles? The tertiary skin and normal size seems what we need here?

Reply:
indeed, I changed it